langchain4j-core/src/main/java/dev/langchain4j/data/audio/Audio.java

```diff
@@ -140,4 +163,19 @@ public Audio build() {
             return new Audio(this);
         }
     }
+
+    /**
+     * Get the filename of the audio using a regex to retrieve the last segment after the last slash.
```

Comment:
I think the word regex isn't really used

Reply:
You want me to call it Regular Expression instead?

Reply:
No

1. You dont use regex to retrieve the last segment
2. You hardcoded the "/" and not use File.Separator (different os)